taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs

```diff
@@ -666,36 +608,27 @@ public void ShowRecorder(object sender, EventArgs e)
 
         public void seleniumAction_SelectionChangeCommitted(object sender, EventArgs e)
```

Comment:
Please follow the PascalCase Naming convention for all event handlers and also update their references.

Reply:
done